### ISSUE TITLE
How to use MedQA-Open

### ISSUE THREAD

**Issue by 5k5000** [ColdSeal/Few-shot-chain-of-thought-driven-reasoning-to-prompt-LLMs-for-open-ended-medical-question-answering#1]
Hi, thanks for your great work '_Few shot chain-of-thought driven reasoning to prompt LLMs for open-ended medical question answering'_. 
Could you please instruct us on how to use MedQA-Open Dataset in your Repo? (I find its current format a bit confusing)

**Comment by ColdSeal** [comment]
Thank you for your appreciation of our work, and we apologize for the delay in responding to your comment. We have updated our GitHub repository to make it more reader-friendly with better structuring and readme. Regarding your query, you can now check the datasets folder containing the different datasets we used in our research and the responses using the four prompting strategies. The medQA-Open dataset is the modified open-ended version of the original MCQ-based MedQA-USMLE dataset, which was intended to mimic real-world settings where the medical practitioner is not provided with any options. The patient history and the type of query asked in the original MCQ version are left unchanged. Only the nature of the query is changed to a descriptive setting, which demands a more rigorous analysis of the given patient history. With the open-ended version, we have also provided the responses using the Llama-2 70B model, wherein we provided the question and the correct answer and prompted the model to provide the supporting reasoning for the correct answer. 
MedQA-Open dataset can be used in multiple ways depending upon the objective of the research:

- The open-ended MedQA-USMLE version can be taken as the basis for performing more rigorous experiments testing the LLMs on challenging settings where the answer space is not confined to four options
- The question and response pair can be used to fine-tune the LLM on an open-ended version of the MedQA-USMLE dataset. We believe this will make the LLM better in responding to complex medical queries where the answer space is not limited and the focus is more on the provided context in answering the query

You may also use the verifier training dataset to train better reward-based models with different base LLMs and strategies.
If you have any more questions, please don't hesitate to ask. We appreciate your taking the time to go through our repository.

**Comment by ColdSeal** [comment]
Closing this issue as it has been resolved.
